manim/mobject/geometry.py

```diff
@@ -336,43 +362,50 @@ def point_at_angle(self, angle):
         return self.point_from_proportion((angle - start_angle) / TAU)
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Dot(Circle):
-    CONFIG = {
-        "radius": DEFAULT_DOT_RADIUS,
-        "stroke_width": 0,
-        "fill_opacity": 1.0,
-        "color": WHITE,
-    }
+    radius: float = DEFAULT_DOT_RADIUS
+    stroke_width: float = 0
```

Comment:
that or change to 0.0
```suggestion
    stroke_width: tp.Union[int, float] = 0
```
```suggestion
    stroke_width: float = 0.0
```